src/appleseed.python/bindblenderexport.cpp

```diff
@@ -84,17 +92,131 @@ namespace
 
     struct MLoopUV
     {
-        float uv[2];
-        int flag;
+        float   uv[2];
+        int     flag;
     };
 
     struct MPoly
     {
-        int loopstart;
-        int totloop;
-        short mat_nr;
-        char flag, pad;
+        int     loopstart;
+        int     totloop;
+        short   mat_nr;
+        char    flag, pad;
+    };
+
+
+    //
+    // This class stores and manipulates an arbitrary number of appleseed transform sequences
+    // from Blender particle and dupli instances.  Storing them in a C++
+    // std::vector allows them to be processed and inserted into the appleseed scene
+    // without the overhead of Python.
+    //
+    
+    class BlTransformLibrary
+    {
+      public:
+        BlTransformLibrary() = default;
```

Comment:
You can remove both the constructor and destructor. By default they are `= default`.